src/components/StatisticsCard/Overview.js

```diff
@@ -5,7 +5,8 @@ import { Card, CardHeader, CardBody, CardImg } from 'reactstrap';
 ChartJS.register(RadialLinearScale, PointElement, LineElement, Filler, Legend, Tooltip);
 
 const OverviewCard = (props) => {
-  const lastSnapshot = props.tankStats.data.snapshots.at(-1);
+  const stats = props.tankStatsCard ? props.tankStatsCard : props.accountStats.data;
```

Comment:
To simplify the code, you can utilize the logical OR (||) operator
```suggestion
  const stats = props.tankStatsCard || props.accountStats?.data;
```